src/main/java/teammates/storage/sqlentity/FeedbackQuestion.java

```diff
@@ -0,0 +1,268 @@
+package teammates.storage.sqlentity;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.questions.FeedbackQuestionType;
+import teammates.common.util.FieldValidator;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Converter;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Inheritance;
+import jakarta.persistence.InheritanceType;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+/**
+ * Represents a FeedbackQuestion
+ *  entity.
+ */
+@Entity
+@Table(name = "FeedbackQuestions")
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+public abstract class FeedbackQuestion extends BaseEntity {
+    @Id
+    @GeneratedValue
```

Comment:
remove generated value, we will generate this ourselves using java uuid library.

Reply:
removed,I think we need to remove in `notification.java` as well:
<img width="397" alt="image" src="https://user-images.githubusercontent.com/67156011/221097570-69d51387-c34b-463e-a6b5-6dc3b7c9e3ba.png">

Reply:
Thanks for the catch, will remove.